File: data/imported/knowledge/cells/typescript/merged/snippets/snippet_012.ts
async function testHtmlExport() {
  console.log("Testing HTML export...");

  const workbook = new Workbook();
  const worksheet = workbook.worksheets[0]!;

  worksheet.putValue("A1", "Name");
  worksheet.putValue("B1", "Age");
  worksheet.putValue("C1", "City");
  worksheet.putValue("A2", "Alice");
  worksheet.putValue("B2", 25);
  worksheet.putValue("C2", "New York");
  worksheet.putValue("A3", "Bob");
  worksheet.putValue("B3", 30);
  worksheet.putValue("C3", "London");

  const html = worksheetToHtml(worksheet);
  console.log("HTML:\n", html);

  await workbook.save("outputfiles/test_html_export.xlsx");
  console.log("Saved to outputfiles/test_html_export.xlsx");
}

File: data/imported/knowledge/cells/rust/merged/snippets/snippet_012.rs
fn main() -> Result<(), Box<dyn Error>> {
    let mut workbook = Workbook::new();
    {
        let mut worksheets = workbook.get_worksheets_mut();
        let sheet = worksheets.get(0)?;
        sheet.set_name("Style Gallery")?;

        let mut cells = sheet.get_cells_mut();
        configure_layout(&mut cells)?;
        write_title(&mut cells)?;
        write_font_section(&mut cells)?;
        write_fill_section(&mut cells)?;
        write_border_section(&mut cells)?;
        write_alignment_section(&mut cells)?;
        write_number_format_section(&mut cells)?;
        write_misc_section(&mut cells)?;
    }

    let output_path = common::output_path("styles-gallery-sample.xlsx");
    workbook.save(&output_path)?;
    println!("Saved: {}", output_path.display());

    let sheet = workbook.worksheet("Style Gallery")?;
    let cells = sheet.get_cells();
    print_summary(&cells)?;

    Ok(())
}